src/components/MovieCard.vue

```diff
@@ -50,10 +52,20 @@ export default {
     margin: 15px;
     font-family: $ff-primary;
     color: $cl-dark;
+    cursor: pointer;
+    transition: 0.5s;
```

Comment:
We should specify which property we want to animate: for example `transition: color 0.5s`. Not setting it will produce `transition: all` which as far as I know is bad for performance. We don't want to animate everything, in most cases it will be one or two properties. 😉